src/test/java/com/team701/buddymatcher/controllers/pairing/PairingControllerIntegrationTest.java

```diff
@@ -49,22 +49,22 @@ AddBuddyDTO createMockedAddBuddyDTO(String userId, String buddyId) {
 
     @Test 
     void removeValidBuddy() {
-        String userId = UUID.randomUUID().toString();
-        String buddyId = UUID.randomUUID().toString();
+        Long userId = new Random().nextLong();
+        Long buddyId = new Random().nextLong();
 
         RemoveBuddyDTO buddyRequest = createMockedRemoveBuddyDTO(userId, buddyId);
 
         ResponseEntity response = pairingController.removeBuddy(buddyRequest);
 
         //Temp response
-        String success = String.format("\"Successfully Removed: %s, %s \"", userId, buddyId);
+        String success = String.format("\"Removed: %s, %s \"", userId, buddyId);
```

Comment:
This test was failing due to the additional "Successfully". PairingController does not contain the word "Successfully"